### DISCUSSION TITLE
minio diff json produces diff code 7 ( not doucmented)

### DISCUSSION THREAD

**Discussion by billmetangmo** [minio/minio#12998] [Q&A] [answered]
Hello,

I have 2 S3 locations loc1 and loc2 configured with minio client (mc) through mc config add.
I decided to do a diff between two folders in these locations.
`mc --json diff loc/1/ loc/4/`

This produce the following result:
```
{
 "status": "success",
 "first":  "https://loc/1/60180877c636430e54452a79/",
 "second": "https://loc/4/60180877c636430e54452a79/",
 "diff": 7
}
```
However i don't know to which difference correspond the number 7 as it's not documented in the documentation https://docs.min.io/docs/minio-client-complete-guide.html#diff as you can see in the atteched picture.
![image](https://user-images.githubusercontent.com/25366207/129929601-5187aeea-6226-46b6-b37d-c299b68af71c.png)
i

**Comment by klauspost** [comment, marked as answer]
https://github.com/minio/mc/pull/3778 updates the table.

**Reply by billmetangmo** [reply]
Thanks.
